packages/fast-web-utilities/src/dom.ts

```diff
@@ -55,10 +55,17 @@ export function canUseCssGrid(): boolean {
     return _canUseCssGrid;
 }
 
-export function canUsedForcedColors(): boolean {
+export function canUseForcedColors(): boolean {
     return (
         canUseDOM() &&
         (window.matchMedia("(forced-colors: none)").matches ||
             window.matchMedia("(forced-colors: active)").matches)
     );
 }
+
+/**
+ * @deprecated Use 'canUseForcedColors' instead
+ */
+export function canUsedForcedColors(): boolean {
```

Comment:
you can simply assign this - 

`export const canUsedForcedColors: typeof canUseForcedColors = canUseForcedColors;`

Reply:
updated